modules/core/shared/src/main/scala/lucuma/core/model/SourceProfile.scala

```diff
@@ -42,14 +43,20 @@ sealed trait SourceProfile extends Product with Serializable {
 
 object SourceProfile {
 
+  /**
+   * Point source.
+   *
+   * @param spectralDefinition
+   *   integrated spectral definition
+   */
   final case class Point(spectralDefinition: SpectralDefinition[Integrated]) extends SourceProfile {
 
     override def toPoint: Point = this
 
     override def toUniform: Uniform =
       Uniform(spectralDefinition.to[Surface])
 
-    override def toGaussian: Gaussian = Gaussian(GaussianSource.Zero, spectralDefinition)
+    override def toGaussian: Gaussian = Gaussian(Angle.Angle0, spectralDefinition)
```

Comment:
I didn't know you could convert them, I guess it makes sense that a point is gaussian with 0 width